### DISCUSSION TITLE
Is there a live GCode-View?

### DISCUSSION THREAD

**Discussion by dasarne** [winder/Universal-G-Code-Sender#2090] [Help]
Hi,
is there any way to show the line of GCode that is currently executing live in the Source-View while milling?  
That would be nice to get a deeper understanding of what's happening in the GCode.

Yours
 Arne

**Comment by chrisla23** [comment]
You can watch the current line being sent in the console view. They will
whiz by pretty fast though.

-Chris

**Reply by dasarne** [reply]
Thank you for the answer. 
You mean the output that shows what was just sent to the mill? That is a good idea. Unfortunately I don't see what is about to come and if I want to change something at that point it will be time consuming.
The line that is being milled must be known in the core of UGS. So one could set the component that shows the source code to that line. Many programs have a switch for something like this. It could be called something like "live view on/off". It might be a low hanging fuite...

**Reply by lamikr** [reply]
UGS classic is showing the information how many lines are in total, how many are now executed and how many still left. It would be nice to get that information showed also to ugs platform.
